services/ansibler/templates/conf.gotpl

```diff
@@ -9,8 +9,7 @@ upstream {{ $role.Role.Name }}{
 server  {
     listen {{ $role.Role.Port }};
     proxy_pass {{ $role.Role.Name}};
-    proxy_protocol on;
```

Comment:
Shouldn't this line be part of the `listen` statement ?

`listen {{ $role.Role.Port }} proxy_protocol`

I think 
`proxy_protocol on;` on its own does something else.